db/bigtable_eth1.go

```diff
@@ -2123,31 +2123,39 @@ func (bigtable *Bigtable) GetInternalTransfersForTransaction(transaction []byte,
 	}
 
 	names := make(map[string]string)
-	// init
-	for _, t := range transfers {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
-	}
+	{
+		forNames := make(map[string]string)
+		for _, t := range transfers {
+			forNames[string(t.From)] = ""
+			forNames[string(t.To)] = ""
+		}
 
-	g := new(errgroup.Group)
-	g.SetLimit(25)
-	for address := range names {
-		address := address
-		g.Go(func() error {
-			name, err := bigtable.GetAddressName([]byte(address))
-			if err != nil {
-				return err
+		g := new(errgroup.Group)
+		g.SetLimit(25)
+		for address := range forNames {
+			address := address
```

Comment:
I know this was not changed by you, but the naming of this variable is VERY confusing (as the name is already used).
Feel free to change or to ignore.

Reply:
Yes, I agree, but this seems to be the Golang way of doing it. :man_shrugging: